pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "tonic-validate"
-version = "4.0.2"
```

Comment:
FYI we shouldn't do the version bump until we release, but I think it's fine for now to do it in this pr